### PR TITLE
Fixes multiple orbiting blades being used at once

### PR DESCRIPTION
**Pull request by SmArtKar**
## About The Pull Request

Fixes the second part of #66784

## Changelog

:cl:
fix: Multiple orbiting blades will no longer be used to protect you from the same attack in case you have multiple instances of the status effect
/:cl:

### CODE REVIEW

**Comment by SmArtKar** [comment]
btw there's no way to do this without using callback, simple REMOVE_TRAIT doesn't solve the issue

**Comment by tralezab** [comment]
@MrMelbert

File: code/datums/status_effects/buffs.dm [hunk 1]
```diff
@@ -528,6 +533,8 @@
 
 	qdel(to_remove)
 
+	addtimer(TRAIT_CALLBACK_REMOVE(source, TRAIT_BEING_BLADE_SHIELDED, type), 1)
```
Comment (tralezab) [inline code/datums/status_effects/buffs.dm:536]: (point of suck)

**Review by tralezab — APPROVED** [review]
it's not... i think we all agree that this kinda sucks but at least the suck is contained to a single line.

**Pull request merged**